sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -668,7 +668,8 @@
                 "supportsHttpsTrafficOnly": true,
                 "allowBlobPublicAccess": false,
                 "networkAcls": "[if(equals(parameters('storageAccountBehindVNet'), 'true'), variables('networkRuleSetBehindVNet'), json('null'))]",
-                "isHnsEnabled": "[equals(parameters('set_up_feature_store'), 'true')]"
+                "isHnsEnabled": "[equals(parameters('set_up_feature_store'), 'true')]",
+                "minimumTlsVersion": "TLS1_2"
```

Comment:
@seanyao1 , we make the storage TLS version 1.2. It also applies to the default storage. If it is not ok, we will make it only for feature store default storage.

Reply:
let's enforce this only for FS storage